### ISSUE TITLE
[Bug Report] Difficulty tooltip showing

### ISSUE THREAD

**Issue by alvaroglezeri** [garciafdezpatricia/TidyTime#28]
In the task list view, if you hover over the difficulty icons, a tooltip is shown that says 'difficulty'. I suppose that this is not the intended behavior.

**Comment by garciafdezpatricia** [comment]
Yes, it was. The idea was to let the user know that those icons were showing the difficulty of the tasks. However, do you have a suggestion in mind? @alvaroglezeri
